helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/AssignableNode.java

```diff
@@ -51,10 +51,9 @@
   private Map<String, Integer> _maxCapacity;
   private int _maxPartition; // maximum number of the partitions that can be assigned to the node.
 
-  // proposed assignment tracking
-  // <resource name, partition name set>
-  private Map<String, Set<String>> _currentAssignments;
-  // <resource name, top state partition name>
+  // A map of <resource name, <partition name, state>> to track all the proposed partition assignment with state.
```

Comment:
partition name? Do you mean the instance's name?

Reply:
What you see is correct. We are already in a node class. No need to put instance name.